consensus/consensus_service.go

```diff
@@ -199,7 +199,7 @@ func (consensus *Consensus) String() string {
 	}
 
 	return fmt.Sprintf(
-		"[Duty:%s Pub:%s Header:%s Num:%d View:%d Shard:%d Epoch:%d]",
```

Comment:
Remove this whole string method please , it has race condition and it’s not needed

Reply:
Done.